tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -532,7 +532,18 @@ public void processTonyConfResources(Configuration tonyConf, FileSystem fs) thro
         continue;
       }
       for (String resource: resources) {
-        LocalizableResource lr = new LocalizableResource(resource, fs);
+        LocalizableResource lr;
+        // If a path does not exist, skip rather than failing only for hdfs classpaths.
```

Comment:
Can you elaborate on the motivation of this change for hdfs classpath in the comment and the PR description?